net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,9 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+int kNonAddableResourceHeadersSize = 12;
```

Comment:
You don't really need this or the DCHECK which is the only place it's referenced.